sklearn/pipeline.py

```diff
@@ -53,12 +53,16 @@ def check(self):
 
 class Pipeline(_BaseComposition):
     """
-    Pipeline of transforms with a final estimator.
+    A sequence of data transformers, which can be optionally complemented with
+    a final predictor.
```

Comment:
The [CI](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=57493&view=logs&j=78a0bf4f-79e5-5387-94ec-13e67d216d6e&t=f1857171-4a53-55c7-3ab5-90acfe091baa) is failing because we are following NumPyDoc standards which require the "summary line" to fit one line.